concurrent/actor/Actor.java

```diff
@@ -44,14 +43,9 @@ protected State(Actor<STATE> self) {
         protected <CHILD_STATE extends State<CHILD_STATE>> Actor<CHILD_STATE> child(Function<Actor<CHILD_STATE>, CHILD_STATE> stateConstructor) {
             return child(self.eventLoop, stateConstructor);
         }
-
-        protected Actor<STATE> self() {
```

Comment:
Let's not remove this because this is needed in reasoner.

Reply:
Cool will add this back.